server/spec/policies/user_policy_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe UserPolicy do
+  let(:user) { build(:user) }
+  let(:admin) { create(:user, :admin) }
+  let(:other) { build(:user) }
+  subject { described_class }
+
+  permissions :show? do
+    it ('should allow anons') { should permit(nil, other) }
+    it ('should allow users') { should permit(user, other) }
+  end
+
+  permissions :create? do
+    it ('should allow admins') { should permit(admin, other) }
+    it ('should allow normal users') { should permit(user, other) }
```

Comment:
This should probably actually be denied, now that I look at it in spec form. Users shouldn't be able to make other users, that's just weird